salt/cloud/clouds/gce.py

```diff
@@ -2616,6 +2623,9 @@ def update_pricing(kwargs=None, call=None):
     url = 'https://cloudpricingcalculator.appspot.com/static/data/pricelist.json'
     price_json = http.query(url, decode=True, decode_type='json')
 
+    if not os.path.exists(__opts__['cachedir']):
```

Comment:
I'd be OK with that, but it should be considered that, at least for my use case, there really isn't much of a "higher up" in terms of the stack. I'm directly using `salt.cloud.Client()`, so there's basically one or two functions between here and my calling code. 

Where should it go?